tensorflow/contrib/cmake/tf_tests.cmake

```diff
@@ -153,7 +153,12 @@ if (tensorflow_BUILD_PYTHON_TESTS)
     "${tensorflow_source_dir}/tensorflow/contrib/data/*_test.py"
     "${tensorflow_source_dir}/tensorflow/contrib/factorization/*_test.py"
     "${tensorflow_source_dir}/tensorflow/contrib/image/*_test.py"
-    "${tensorflow_source_dir}/tensorflow/contrib/keras/python/keras/integration_test.py"
+    "${tensorflow_source_dir}/tensorflow/python/keras/_impl/keras/*_test.py"
```

Comment:
I think (by the magic of `GLOB_RECURSE`) this line implies lines 157 through 161 and you can delete them.

Reply:
Done